modules/installation-aws-config-yaml.adoc

```diff
@@ -82,3 +82,8 @@ provides the default value.
 machines in your cluster.
 <4> To configure faster storage for etcd, especially for larger clusters, set the
 storage type as `io1` and set `iops` to `2000`.
+<5> For a CIDR in the range of 10.128.0.0/14, IPs in this range are, by default,
+ assigned to the Pods. If you want to change this, you must adjust the
```

Comment:
What would you change this to, and how would a user know what to change to get what they wanted?

Reply:
@Deepthidharwar Any suggestions?

Reply:
It's basically a  block of IP addresses one reserves, from which Pod IPs are allocated. If one wants Pod IPs to be different from what is reserved by default, then one needs to modify this.